sw/device/tests/BUILD

```diff
@@ -3593,6 +3593,28 @@ cc_library(
     ],
 )
 
+opentitan_test(
+    name = "spi_device_ottf_console_test",
+    srcs = ["spi_device_ottf_console_test.c"],
+    exec_env = dicts.add(
+        EARLGREY_CW340_TEST_ENVS,
```

Comment:
can you add the first two exec environments here too (the `fpga_cw310*` ones): https://cs.opensource.google/opentitan/opentitan/+/master:rules/opentitan/defs.bzl;drc=8fdb25c8d96f8a7b267a3344058299b127278765;l=90 ?

Reply:
Added. Since I don't have cw310 environment to run the test in advance, let's see if the test works as expected on the CI pipeline.

Reply:
Looks like the test works as expected on cw310 https://dev.azure.com/lowrisc/opentitan/_build/results?buildId=158483&view=logs&j=6d7ef521-d3a1-575b-de1b-a7342dcf1a8e&t=5fbf7d75-9dee-5f42-7084-95893df8eb52&l=14748

Some failures appear to be unrelated to this PR, though
https://github.com/lowRISC/opentitan/pull/24345/checks?check_run_id=29048541424
Regarding the error in private CI, I am not sure if it's related to this PR because I don't have permission to access private CI.

Reply:
I re-trigger CI; hopefully these should pass now.